### ISSUE TITLE
Wrap condition into '()' to specify intention: not need for spaceship operation

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#868]
The following code is warning "_This may not work as expected (wrap condition into '()' to specify intention)_" over the spaceship operation. The spaceship precedence is greater than `?:`, so it is not need here.

```php
   $a <=> $b ?: $c;
// ^^^^^^^^^
```

**Comment by kalessil** [comment]
Understood, will fix in the next release.

**Comment by kalessil** [comment]
@rentalhost : re-visited the issue, the `()` is making sense - the intention is not clear as for me.

**Comment by rentalhost** [comment]
Then it actually is a style inspection, not a possible issue on code?
